tools/testbench/testbench.c

```diff
@@ -637,7 +651,11 @@ static int pipline_test(struct testbench_prm *tp)
 				dp_count, err);
 			break;
 		}
+#if defined __XCC__
```

Comment:
Can we abstract these xtensa APIs to a high level ```tb_gettime()``` etc APIs that calls either the xtensa or host versions. This will keep all the XCC/host changes at the top of the file